docker-compose.yml

```diff
@@ -1,56 +0,0 @@
-version: "3.7"
```

Comment:
why remove?

Reply:
Yeah didn't mean to. I had accidentally commit my local copies that had the volumes mounted, in docker-compose and my own api-key in config.json, then when I tried to remove my accidental commit from this PR it lead to this mess.. Haha.

Reply:
No worries, just delete the two files from your PR and ill merge it

Reply:
Okay so I restored those two files back to the state of the upstream that I forked from, so I'm not sure why theres those few small differences in there now.